webpack.config.js

```diff
@@ -12,33 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const path = require('path');
+
 module.exports = {
-  entry: './src/browser.js',
+  entry: './src/index.ts',
   output: {
-    library: 'iot',
-    filename: './iot.js',
+    library: 'DeviceManager',
```

Comment:
This is breaking! (even though nobody uses it)

Let's use our cool new option `--main-service iot` here.